<launch>

<rosparam command="load"  file="$(find pointcloud_filter)/config/kalman_params.yaml" />
  <node name="pc_filter" pkg ="pointcloud_filter" type="pointcloud_filter"/>
</launch>

